extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -377,7 +433,7 @@ export class SKURecommendationPage extends MigrationWizardPage {
 		if (this.migrationStateModel._assessmentResults) {
 			const dbCount = this.migrationStateModel._assessmentResults.databaseAssessments.length;
 
-			const dbWithIssuesCount = this.migrationStateModel._assessmentResults.databaseAssessments.filter(db => db.issues.length > 0).length;
+			const dbWithIssuesCount = this.migrationStateModel._assessmentResults.databaseAssessments.filter(db => db.issues.length === 0).length;
```

Comment:
The name doesn't seem to match what's being done anymore - you're filtering it to just assessments with NO issues.

Reply:
I am getting the count of database with no assessment issues.

Reply:
Then the name should be `dbWithoutIssuesCount` 😄